.github/pull_request_template.md

```diff
@@ -1,26 +1,69 @@
+> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*
 
-### This pull request is ready to merge when...
+> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*
 
-- [ ] Meets acceptance criteria for the issue 
-- [ ] Code is reviewed by someone other than the original author
-- [ ] Code is tested 
-- [ ] Experience is approved by UX  
-- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
-- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
-- [ ] Documentation is updated 
-  - OpenAPI  
-  - Readme
-  - Entity Relationship Diagram (ERD)
++ Are all areas of code introduced in this PR meaningfully tested?
+  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
+  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
++ Are code coverage minimums met?
+  + [ ] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
+  + [ ] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
+
+### Deliverable 3: Properly Styled Code
+
+> *Performance Standard(s): GSA 18F Front- End Guide*
+
+> *Acceptable Quality Level: 0 linting errors and 0 warnings*
+
++ [ ] Are backend code style checks passing on CircleCI?
++ [ ] Are frontend code style checks passing on CircleCI?
++ [ ] Does this PR change any linting or CI settings?
+
+### Deliverable 4: Accessible
+
+> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*
+
+> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*
+
++ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
+    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
+
+### Deliverable 5: Deployed
+
+> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*
+
+> *Acceptable Quality Level: Successful build with a single command*
+
++ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?
```

Comment:
@carltonsmith correct me if I'm wrong but as I understand it nothing currently gets deployed to the development instance until it gets merged in by HHS. If so, we may not be able to complete this particular step during Raft review.

That may change once the staging site is set up, and would probably be better if we could deploy to the dev environment prior to submitting for QASP review so we can isolate any issues ahead of time and prevent needing to rollback the HHS repo.

Reply:
@jtwillis92 Yes, this process changed from when we originally developed this checklist, so you are right, this doesn't happen until after it is merged to HHS. We do have a way of launching an automated deployment when we merge with `raft-review` but I need to find a cleaner way of doing that, as we can often get things in there that aren't part of what we are testing.